search-app/docs/api.md

```diff
@@ -4077,6 +4084,263 @@ Content-Length: 168
 
 On occasions when service dissociation cannot be processed at all due to invalid input, service dissociation request will return a failure status code with the appropriate error message.
 
+### <a name="subscription"></a> Subscription
+
+A subscription enables data to be accessed via a universal resource locator, Subscription metadata is in JSON format and conforms to [UMM-Sub Schema](https://git.earthdata.nasa.gov/projects/EMFD/repos/unified-metadata-model/browse/subscription).
```

Comment:
Can we add more text here to indicate what is a subscription and how it is supposed to be used?

Reply:
Yes we can.